### PR TITLE
Fix registry setup

### PR DESCRIPTION
**Pull request by nocturnalastro**

### CODE REVIEW

**Review by mkochanowski — DISMISSED** [review]

File: inventory.yml.sample [hunk 1]
```diff
@@ -115,7 +115,7 @@ all:
     ssh_public_key: "{{ lookup('file', local_ssh_public_key_path) }}"
 
     # provided mirror certificate logic, no need to change.
-    local_mirror_certificate_path: "{{ (setup_registry_service == true) | ternary(fetched_dest + '/domain.crt',  repo_root_path + '/mirror_certificate.txt') }}"
+    local_mirror_certificate_path: "{{ (setup_registry_service == true) | ternary(fetched_dest + '/registry.crt',  repo_root_path + '/mirror_certificate.txt') }}"
```
Comment (arjuhe) [inline inventory.yml.sample:118]: Should `registry.crt` be changed to `{{ cert_file_prefix }}`?

File: roles/setup_mirror_registry/tasks/retrieve_config.yml [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
+- name: "Read in the contents of {{ cert_file_prefix }}.crt"
   slurp:
-    src: "{{ registry_dir_cert }}/domain.crt"
+    src: "{{ registry_dir_cert }}/{{ cert_file_prefix }}.crt"
   register: domain_cert_b64
   tags:
     - copy_config
 
-- name: Set trustbundle fact to contents of domain.crt
+- name: "Set trustbundle fact to contents of {{ cert_file_prefix }}.crt"
```
Comment (arjuhe) [inline roles/setup_mirror_registry/tasks/retrieve_config.yml:17]: Thanks for fixing.

**Review by arjuhe — CHANGES_REQUESTED** [review]

**Review by arjuhe — DISMISSED** [review]
Thanks for making the change. (: +1

File: README.md [hunk 3]
```diff
@@ -177,7 +177,7 @@ The following are defaults for a full setup:
 As well as deploying prerequisites and a cluster, the playbooks create or update various local artifacts in the repository root and the `fetched/` directory (configured with `fetched_dest` var in the inventory).
 
 - An updated `pull-secret.txt` containing an additional secret to authenticate with the deployed registry.
-- The self-signed certificate created for the registry host as `domain.crt`.
+- The self-signed certificate created for the registry host as `registry.crt`.
```
Comment (arjuhe) [inline README.md:180]: +1 Very nice.

**Review by arjuhe — APPROVED** [review]
+1

**Review by mkochanowski — APPROVED** [review]

**Pull request merged**